var/spack/repos/builtin.mock/packages/perl/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Perl(Package):
```

Comment:
Is this package definition necessary given `builtin_and_mock_packages`?

Reply:
It's necessary for other tests that use only the mock repo. Otherwise we get an error from loading the perl-extension package in unrelated tests, since some features require looking at every package.

Reply:
As in, because a mock Perl extension is now included, there must also now be a mock Perl, for tests which just use the builtin repo (vs. builtin + mock)?

Reply:
Correct